src/drawers/drawer.ts

```diff
@@ -76,12 +87,16 @@ export module Drawers {
      *
      * @param{AppliedDrawStep} step The step, how data should be drawn.
      */
-    protected _drawStep(step: Drawers.AppliedDrawStep) {
-      // no-op
-    }
-
-    protected _numberOfAnimationIterations(data: any[]): number {
-      return data.length;
+    private _drawStep(step: Drawers.AppliedDrawStep) {
+      var selection = this._selection();
+      var colorAttributes = ["fill", "stroke"];
+      colorAttributes.forEach((colorAttribute) => {
+        if (step.attrToAppliedProjector[colorAttribute] != null) {
+          selection.attr(colorAttribute, step.attrToAppliedProjector[colorAttribute]);
+        }
```

Comment:
We might consider (later) moving this logic to the `Plot`, in case users want the `stroke` or `fill` to animate.